src/styles/_card.scss

```diff
@@ -0,0 +1,113 @@
+@import 'variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $width-card;
+  color: $color-background;
```

Comment:
The variable `$color-background` is used for the `color` property, which might not be appropriate. Typically, the `color` property is used for text color, so ensure that `$color-background` is intended for text color.